app/serializers/flight_serializer.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class FlightSerializer < ActiveModel::Serializer
+  attribute :fare_category
+
+  attribute :price do
+    object&.formatted_price
+  end
```

Comment:
O que você acha de substituir a sintaxe acima para abaixo?

```suggestion
  attribute :formatted_price, key: :price
```

Reply:
Ajustado em: https://github.com/Laryssa-lfa/flight-company/pull/5/commits/593a768714615351e3a8856b837ae096c6920c62.